permissive-json-pointer/src/lib.rs

```diff
@@ -206,7 +206,7 @@ fn create_value(value: &Document, mut selectors: HashSet<&str>) -> Document {
     new_value
 }
 
-fn create_array(array: &Vec<Value>, selectors: &HashSet<&str>) -> Vec<Value> {
+fn create_array(array: &[Value], selectors: &HashSet<&str>) -> Vec<Value> {
```

Comment:
Unrelated to the issue this PR fixes, but Clippy warns about this:

```
warning: writing `&Vec<_>` instead of `&[_]` involves one more reference and cannot be used with non-Vec-based slices
   --> permissive-json-pointer/src/lib.rs:209:24
    |
209 | fn create_array(array: &Vec<Value>, selectors: &HashSet<&str>) -> Vec<Value> {
    |                        ^^^^^^^^^^^ help: change this to: `&[Value]`
    |
    = note: `#[warn(clippy::ptr_arg)]` on by default
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#ptr_arg
```

Let me know if I should remove this from the PR.

Reply:
that ok, thanks